docs/data/date-pickers/localization/localization.md

```diff
@@ -131,3 +131,10 @@ You can [find the source](https://github.com/mui/mui-x/tree/HEAD/packages/x-date
 
 To create your own translation or to customize the English text, copy this file to your project, make any changes needed and import the locale from there.
 Note that these translations of the date and time picker components depend on the [Localization strategy](/material-ui/guides/localization/) of the whole library.
+
+## Access the translations in slots and subcomponents
+
+You can use the `useLocaleText` hook to access the translations in your custom components.
+The demo below shows how to use the `localeText.okButtonLabel` translation key in a custom Action Bar:
+
+{{"demo": "UseLocaleText.js", "defaultCodeOpen": false}}
```

Comment:
Is it worth duplicating these demos?
WDYT about adding a link to the [Custom slots and subcomponents—Action bar](/x/react-date-pickers/custom-components/#component)? 🤔 

```suggestion

:::info
See [Custom slots and subcomponents—Action bar](/x/react-date-pickers/custom-components/#component) for more details.
:::
```

Reply:
I would at least add a code snippet then
I think people should be able to learn how to use `useLocaleText` without going in a section about the ActionBar.

Reply:
Sure, it makes sense, a code snippet is better than only text and minimizes duplication from demo PoV. 👍